lib/main.dart

```diff
@@ -1,10 +1,14 @@
 import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'screen/home_screen.dart';
+import 'screen/login_screen.dart';
 import 'themes/themes.dart';
-import 'widgets/my_navigation_bar.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+import '../constants.dart';
 
-void main() {
+void main() async {
+  await dotenv.load(fileName: ".env");
```

Comment:
What is this for? If it's just for loading some (non-secret) constants, I think we should just hardcode them on the frontend side. If it's also loading secrets, that's sus. Looks like it's the former. I think we should just move those to `lib/constants.dart`, even the two need to be in sync with the backend.